src/app/pages/candidate-details/__tests__/candidate-details.page.spec.ts

```diff
@@ -134,12 +137,4 @@ describe('CandidateDetailsPage', () => {
     });
   });
 
-  describe('dismiss', () => {
-    it('should dismiss open modal', fakeAsync(async () => {
-      spyOn(component.modalController, 'dismiss');
-      await component.dismiss();
-      expect(component.modalController.dismiss).toHaveBeenCalled();
-    }));
-  });
```

Comment:
Is this not still relevant? We could possibly check also for the ```store$.dispatch``` in the then block

Reply:
reinstated test and updated to reflect code change